Sources/SwiftSyntaxBuilder/ExprConvenienceInitializers.swift.gyb

```diff
@@ -0,0 +1,91 @@
+%{
+  from gyb_syntax_support import *
+  from gyb_syntax_support.kinds import SYNTAX_BASE_KINDS, lowercase_first_word, syntax_buildable_child_type
+  NODE_MAP = create_node_map()
+  # -*- mode: Swift -*-
+  # Ignore the following admonition it applies to the resulting .swift file only
+}%
+//// Automatically Generated From ExprConvenienceInitializers.swift.gyb.
+//// Do Not Edit Directly!
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+
+% for node in SYNTAX_NODES:
+%   if node.is_buildable() and node.base_kind is 'Expr':
+%     has_syntax_collection_child = False
+%     has_token_as_child = False
+%     for child in node.children:
+%       child_node = NODE_MAP.get(child.syntax_kind)
+%       if child_node and child_node.is_syntax_collection():
+%         has_syntax_collection_child = True
+%       end
+%       if SYNTAX_TOKEN_MAP.get(child.syntax_kind) or child.syntax_kind is "IdentifierToken":
+%         has_token_as_child = True
+%       end
+%     end
+%     if not has_syntax_collection_child and has_token_as_child:
```

Comment:
I haven’t looked at this in detail but wouldn’t we get the same (or something similar) if we dropped the `has_syntax_collection_child` requirement from [`BuildablesConvenienceInitializers.swift.gyb`](https://github.com/apple/swift-syntax/blob/18880228fa4de8c3368d66ead6f4e0dfd4c7f1f9/Sources/SwiftSyntaxBuilder/BuildablesConvenienceInitializers.swift.gyb#L33)?

Reply:
I have removed the file and made `BuildablesConvenienceInitializers.swift.gyb` generate more.